lib/error-matcher.js

```diff
@@ -23,12 +23,21 @@ module.exports = (function () {
   util.inherits(ErrorMatcher, EventEmitter);
 
   ErrorMatcher.prototype._goto = function () {
-    var file = this.cwd + path.sep + this.currentMatch[0].file;
-    if (!this.currentMatch[0].file) {
+    var file = this.currentMatch[0].file;
+    if (file) {
+      var oldDir = process.cwd();
+      process.chdir(this.cwd);
```

Comment:
Well, it seems weird if `this.cwd` and `process.cwd()` differs AND the build tool outputs a file relative to `process.cwd()` when the working directory is set to `this.cwd`. Essentially, the executed build command knows nothing of `process.cwd()`.